commcare_connect/opportunity/views.py

```diff
@@ -929,8 +929,15 @@ def reject_visits(request, org_slug=None, opp_id=None):
 
 
 @org_member_required
-def fetch_attachment(self, org_slug, blob_id):
+def fetch_attachment(request, org_slug, blob_id):
     blob_meta = BlobMeta.objects.get(blob_id=blob_id)
+
+    if not UserVisit.objects.filter(
+        opportunity__organization__slug=org_slug,
+        xform_id=blob_meta.parent_id,
+    ).exists():
+        return HttpResponseForbidden()
```

Comment:
We might need to do a program manager and network manager check here. We have these checks in other views using the `get_opportunity_or_404` method or the `@opportunity_required` decorator, both of these are viable methods to check for permissions to access opportunity on a request user.